packages/web/src/components/search/DataSearch.js

```diff
@@ -1001,13 +979,11 @@ const mapStateToProps = (state, props) => ({
 			&& state.selectedValues[props.componentId].value)
 		|| null,
 	suggestions: state.hits[props.componentId] && state.hits[props.componentId].hits,
-	aggregationData: state.compositeAggregations[props.componentId] || [],
+	aggregationData: state.compositeAggregations[props.componentId],
```

Comment:
@jyash97 Same here

Reply:
This is causing an extra re-render because of the fallback.

Reply:
@jyash97 I'm not sure if @ShahAnuj2610 handled it well.
@ShahAnuj2610 Can you confirm if this change wouldn't break the things?